pkg/ocm/versions.go

```diff
@@ -88,6 +89,16 @@ func HasSTSSupport(rawID string, channelGroup string) bool {
 	return a.GreaterThanOrEqual(b)
 }
 
+func HasSTSSupportMinor(minor string) bool {
+	a, erra := ver.NewVersion(minor)
+	b, errb := ver.NewVersion(LowestSTSMinor)
+	if erra != nil || errb != nil {
+		return minor >= LowestSTSMinor
+	}
+
+	return a.GreaterThanOrEqual(b)
```

Comment:
semver has a nice function for this https://pkg.go.dev/github.com/masterminds/semver#Version.GreaterThan

Reply:
I'm using https://github.com/hashicorp/go-version instead. I have no strong opinion on either, but the OCM API already uses the hashicorp lib anyway.